src/cursor/change_stream_cursor.ts

```diff
@@ -55,7 +55,7 @@ export class ChangeStreamCursor<
     pipeline: Document[] = [],
     options: ChangeStreamCursorOptions = {}
   ) {
-    super(client, namespace, options);
+    super(client, namespace, { ...options, tailable: true, awaitData: true });
```

Comment:
Surprising these weren't set before but also why now?

Reply:
Setting those here allows the `AbstractCursor` constructor to do the appropriate validation and to correctly determine whether or not we send the `maxTimeMS` on the command

Reply:
Until CSOT, tailable / awaitData didn't impact the actual cursor commands we send to the server.  So it was implicitly a tailable await data cursor I guess, and our `Cursor.next()` logic works regardless of the cursor's configuration (we rely on the server to tell us when to stop iterating). :shrug